src/garage/torch/policies/tanh_gaussian_mlp_policy.py

```diff
@@ -0,0 +1,72 @@
+"""TanhGaussianMLPPolicy."""
+import torch
+
+from garage.torch.modules import TanhGaussianMLPTwoHeadedModule
+from garage.torch.policies import Policy
+
+
+class TanhGaussianMLPPolicy(Policy, TanhGaussianMLPTwoHeadedModule):
+    """
+    GaussianMLPPolicy.
+
+    A policy that contains a MLP to make prediction based on a gaussian
+    distribution.
+
+    Args:
+        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+        module : GaussianMLPModule to make prediction based on a gaussian
+        distribution.
+    :return:
+    """
+
+    def __init__(self, env_spec, **kwargs):
+        self._obs_dim = env_spec.observation_space.flat_dim
+        self._action_dim = env_spec.action_space.flat_dim
+
+        Policy.__init__(self, env_spec)
+        TanhGaussianMLPTwoHeadedModule.__init__(self,
+                                   input_dim=self._obs_dim,
```

Comment:
Can we make input_dim (maybe and output_dim) configurable parameters? PEARL uses tanh_gaussian_mlp_policy and its input_dim needs to be obs_dim + some number denoting latent dimension. I think this also adds more flexibility in general for input and output dimensions.